From: Ben Hutchings <ben@decadent.org.uk>
Date: Wed, 14 Mar 2018 01:09:10 +0000
Subject: drm/atomic: Avoid ABI change in 4.15.7
Forwarded: not-needed

The new field drm_crtc_commit::abort_completion is only used in
drm_atomic_helper.c, which is the same place the structure is
allocated.  Therefore we can safely hide it from genksyms and
avoid an ABI change.

---
--- a/include/drm/drm_atomic.h
+++ b/include/drm/drm_atomic.h
@@ -135,6 +135,7 @@ struct drm_crtc_commit {
 	 */
 	struct drm_pending_vblank_event *event;
 
+#ifndef __GENKSYMS__
 	/**
 	 * @abort_completion:
 	 *
@@ -143,6 +144,7 @@ struct drm_crtc_commit {
 	 * the free code to remove the second reference if commit fails.
 	 */
 	bool abort_completion;
+#endif
 };
 
 struct __drm_planes_state {
